src/lib/expression/between_expression.hpp

```diff
@@ -23,4 +34,32 @@ class BetweenExpression : public AbstractPredicateExpression {
   ExpressionPrecedence _precedence() const override;
 };
 
+class BetweenInclusiveExpression : public BaseBetweenExpression {
```

Comment:
Just to stress the point, look at these 26 lines + the corresponding definitions in the cpp - they are just code-noise but have no functional purpose and therefore only put cognitive load on the reader.